src/py21cmfast/src/GenerateICs.c

```diff
@@ -185,8 +193,11 @@ int ComputeInitialConditions(unsigned long long random_seed, struct UserParams *
 
                     // ok, now we can draw the values of the real and imaginary part
                     // of our k entry from a Gaussian distribution
-                    a = gsl_ran_ugaussian(r[omp_get_thread_num()]);
-                    b = gsl_ran_ugaussian(r[omp_get_thread_num()]);
+//                    a = gsl_ran_ugaussian(r[omp_get_thread_num()]);
+//                    b = gsl_ran_ugaussian(r[omp_get_thread_num()]);
+                    a = 1.;
+                    b = -1.;
```

Comment:
is this temporary?

Reply:
Yes, this is just me temporarily turning off the RNG so I can easily compare outputs generated from old 21cmFAST as I import new functions etc.

This is the change I referred to potentially making to alleviate the tests failing owing to rounding errors. In principle, we can add a flag (i.e. `TURN_OFF_RNG`) which turns off all RNG usage in 21cmFAST. Then, all tests can take this, and all tests are run without RNG on. It would also be useful for debugging and developing new code.

I think this would be a useful feature, and should be implemented. I can implement this in a separate branch to push this change through faster if you agree.

Reply:
Yeah, let's add that flag. It can probably just stay as part of this branch since so many changes are going into this. You should try to keep a list of all the additions/changes you make in this branch as you go!

Reply:
@steven-murray, would it make sense to add that flag to #87 instead? Essentially that PR is finished (failing because of the rounding issue). It could be implemented there, update the test data and be pushed to master.

Those changes can then be brought into this branch.